### ISSUE TITLE
Champs complémentaires pour les ateliers

### ISSUE THREAD

**Issue by eheliot12** [CroissanceCommune/autonomie#627]
des champs d'informations sont à ajouter pour les ateliers
* lieu
* description
* l'enseigne qui gère l'atelier
* les ressources : salle, materiel
* des tags : une liste de tag est disponible pour classer les ateliers et faciliter les recherches des membres de la cae

#413 fichiers attachés
* le rattachement à une affaire

**Comment by JocelynDelalande** [comment]
Lesquels de ces champs doivent apparaître sur l'émargement ? (je me pose la question particulièrement pour le champ description).

**Comment by eheliot12** [comment]
la feuille d'émargement doit comporter l'intitulé, le lieu, la date avec heure début et heure de fin.

**Comment by sbodrero** [comment]
@eheliot12 la liste des tags est prédéfinie ?

**Comment by eheliot12** [comment]
non, pas prédéfinie. cela pourrait se faire soit à la volée, soit dans une liste configurée par la CAE
cela pourrait etre aussi le champ action financé, si ce n'est qu'il n'est pas multivaleur
